.github/workflows/main.yml

```diff
@@ -1,12 +1,6 @@
 name: API Manager Build with Test
 
-on:
```

Comment:
revert the change in main.yaml

Reply:
reverted with commit: https://github.com/wso2/carbon-apimgt/pull/10010/commits/83725d07cc8d603f6e94fa10d13f1c421eda7b02